K2Bridge/KustoDAL/HitsMapper.cs

```diff
@@ -23,7 +23,7 @@ public static class HitsMapper
         /// <summary>
         /// Type converter function used to get values from column.
         /// </summary>
-        private static readonly Dictionary<Type, Func<object, object>> Converters = new Dictionary<Type, Func<object, object>>
```

Comment:
If you're modernizing the code, consider using the modern dict syntax:
```
        private static readonly Dictionary<Type, Func<object, object>> Converters = new ()
        {
            [typeof(sbyte)] = value => value is DBNull || value == null ? null : (bool?)((sbyte)value != 0),
            [typeof(SqlDecimal)] = value => value.Equals(SqlDecimal.Null) ? double.NaN : ((SqlDecimal)value).ToDouble(),
            [typeof(Guid)] = value => value is DBNull || value == null ? null : ((Guid)value).ToString(),
            [typeof(TimeSpan)] = value => value is DBNull || value == null ? null : XmlConvert.ToString((TimeSpan)value),

            // Elasticsearch returns timestamp fields in UTC in ISO-8601 but without Timezone.
            // Use a String type to control serialization to mimic this behavior.
            [typeof(DateTime)] = value => value is DBNull ? null : ((DateTime)value).ToString("yyyy-MM-dd'T'HH:mm:ss.FFFFFFF"),
        };
```
Or even further with the "or null" pattern:

```
        private static readonly Dictionary<Type, Func<object, object>> Converters = new ()
        {
            [typeof(sbyte)] = value => value is DBNull or null ? null : (bool?)((sbyte)value != 0),
            [typeof(SqlDecimal)] = value => value.Equals(SqlDecimal.Null) ? double.NaN : ((SqlDecimal)value).ToDouble(),
            [typeof(Guid)] = value => value is DBNull or null ? null : ((Guid)value).ToString(),
            [typeof(TimeSpan)] = value => value is DBNull or null ? null : XmlConvert.ToString((TimeSpan)value),

            // Elasticsearch returns timestamp fields in UTC in ISO-8601 but without Timezone.
            // Use a String type to control serialization to mimic this behavior.
            [typeof(DateTime)] = value => value is DBNull ? null : ((DateTime)value).ToString("yyyy-MM-dd'T'HH:mm:ss.FFFFFFF"),
        };
```

Reply:
thank you @AsafMah - it's done.